plugin/manager.go

```diff
@@ -138,17 +138,21 @@ func (pm *Manager) tmpDir() string {
 // HandleExitEvent is called when the executor receives the exit event
 // In the future we may change this, but for now all we care about is the exit event.
 func (pm *Manager) HandleExitEvent(id string) error {
+	logrus.WithField("id", id).Debug("HandleExitEvent")
```

Comment:
Remove.

Reply:
Why?

Reply:
I assume it was added for development purposes only. Didnt see a real value add when in the field.